install-aws-ofi-nccl.sh

```diff
@@ -12,8 +26,21 @@ git log -1
             --with-cuda=/usr/local/cuda \
             --with-nccl=$HOME/nccl/build \
             --with-mpi=/opt/amazon/openmpi
+if [ $? -ne 0 ]; then
+    echo "Configure failed!"
+    exit -1
```

Comment:
Currently all of these scripts are prepended to single script, which has an `set -e` at the beginning. So this return status check is useless with that setting. (Same for all the other scripts).

Reply:
I understand that. I am writing a new set of scripts to do GDRTest, which will not call `set -e`. Meanwhile, adding this error handling does not have negative impacts, right?

Reply:
Yes, this does not hurt, so I am ok with it.